backend/internal/controller/server/msg_handler.go

```diff
@@ -66,3 +67,26 @@ func (s server) createResponse(reader io.Reader) ([]byte, error) {
 func (s server) ResetScoreHandler(w http.ResponseWriter, r *http.Request) {
 	s.game.ResetScore()
 }
+
+//Incoming URL should be in the format:
+// '/goal?action=[add/sub]&team=[1/2]'
+// team 1 - White; team 2 - Blue
```

Comment:
Maybe use OpenAPI for documenting endpoints?

Reply:
We will probably use it in the future. For now it is not so important for us and we want to focus on other things.